recognition/src/mask_map.cpp

```diff
@@ -68,30 +57,35 @@ pcl::MaskMap::resize (const size_t width, const size_t height)
 
 
 //////////////////////////////////////////////////////////////////////////////////////////////
+
 void
 pcl::MaskMap::getDifferenceMask (const MaskMap & mask0,
                                  const MaskMap & mask1,
                                  MaskMap & diff_mask)
+{
+    diff_mask = getDifferenceMask(mask0, mask1);
```

Comment:
We use two spaces for indentation.

Actually, we have a clang-format style in place, but it hasn't been applied to the entire codebase yet. Perhaps we can already run these files through the formatter as a part of this PR. @kunaltyagi what do you think?

Reply:
Reformatting the entire file will hide the real changes, so probably I should make that change after you will have approved these ones

Reply:
Reformat after formatter has been through. Let's keep whitespace changes to a minimum